script.js

```diff
@@ -1,4 +1,267 @@
+    concastenateNumberPlayer2.push(concastenateNumber4)
+    return `Player 2, you chose Dice 2 first.<br>Your number is ${concastenateNumberPlayer2[0]}.Click submit to determine the result`
+}
+if(userPick==3){
+  if(diceNumberPlayer2[0]>=diceNumberPlayer2[1]){
+    concastenateNumberPlayer2.push(concastenateNumber3)
+    return `Player 1, you chose Dice 1 first.<br>Your number is ${concastenateNumberPlayer2[0]}.<br>Click submit to determine the result`
+  }
+  if(diceNumberPlayer2[1]>=diceNumberPlayer2[0]){
+     concastenateNumberPlayer2.push(concastenateNumber4)
+    return `Player 1, you chose Dice 2 first.<br>Your number is ${concastenateNumberPlayer2s[0]}.<br>Click submit to determine the result`
+  }
+}
+}
+}
+var resultHighest = function(){
+  gameMode='dice roll'
+  currPlayer=1
+  console.log(concastenateNumberPlayer1,concastenateNumberPlayer2)
+  if(concastenateNumberPlayer1[0]>concastenateNumberPlayer2[0]){
+    player1NumberWin+=1
+    diceNumberPlayer1=[]
+    diceNumberPlayer2=[]
+    concastenateNumberPlayer1=[]
+    concastenateNumberPlayer2=[]
+    if(player1NumberWin>player2NumberWin){
+      return `Player 1 win yay<br>1. Player 1 win :${player1NumberWin}<br>2. Player 2 win :${player2NumberWin}`
+    }
+    if(player1NumberWin<player2NumberWin){
+      return `Player 1 win yay<br>1. Player 2 win :${player2NumberWin}<br>2. Player 1 win :${player1NumberWin}`
+    }
+    return `Player 1 win yay<br>1. Player 2 win :${player2NumberWin}<br>1. Player 1 win :${player1NumberWin}`
+  }
+  if(concastenateNumberPlayer1[0]<concastenateNumberPlayer2[0]){
+    player2NumberWin+=1
+    diceNumberPlayer1=[]
+    diceNumberPlayer2=[]
+    concastenateNumberPlayer1=[]
+    concastenateNumberPlayer2=[]
+     if(player1NumberWin>player2NumberWin){
+      return `Player 2 win yay<br>1. Player 1 win :${player1NumberWin}<br>2. Player 2 win :${player2NumberWin}`
+    }
+    if(player1NumberWin<player2NumberWin){
+      return `Player 2 win yay<br>1. Player 2 win :${player2NumberWin}<br>2. Player 1 win :${player1NumberWin}`
+    }
+    return `Player 2 win yay<br>1. Player 2 win :${player2NumberWin}<br>1. Player 1 win :${player1NumberWin}`
+  }
+    diceNumberPlayer1=[]
+    diceNumberPlayer2=[]
+    concastenateNumberPlayer1=[]
+    concastenateNumberPlayer2=[]
+    if(player1NumberWin>player2NumberWin){
+      return `draw<br>1. Player 1 win :${player1NumberWin}<br>2. Player 2 win :${player2NumberWin}`
+    }
+    if(player1NumberWin<player2NumberWin){
+      return `draw<br>1. Player 2 win :${player2NumberWin}<br>2. Player 1 win :${player1NumberWin}`
+    }
+    return `draw<br>1. Player 2 win :${player2NumberWin}<br>1. Player 1 win :${player1NumberWin}`
+}
+// lowest number mode
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////
+var concastenateModeLowest=function(input){
```

Comment:
I am not very sure what concastenate represent. Are you referring to concatenateModeLowest?